contrib/src/main/java/com/datatorrent/contrib/kafka/KafkaConsumer.java

```diff
@@ -86,8 +86,10 @@ public KafkaConsumer(String zks, String topic)
   protected String topic = "default_topic";
 
   /**
-   * A zookeeper map keyed by cluster id
-   * It's mandatory field
+   * A zookeeper map keyed by cluster id <br>
+   * It's mandatory field <br>
+   * Each cluster should have only one connection string contain all nodes in the cluster <br>
+   * Chroot path is also supported
```

Comment:
Also good to mention that this is the syntax as accepted by <ref ZkClient> or zookeeper docs?